extensions/wikia/AdEngine/js/template/bfaa.js

```diff
@@ -0,0 +1,55 @@
+/*global define*/
+define('ext.wikia.adEngine.template.bfaa', [
+	'ext.wikia.adEngine.adHelper',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (adHelper, doc, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.bfaa',
+		// SCSS property: $breakpoint-width-not-supported
+		breakPointWidthNotSupported = 767,
+		nav = doc.getElementById('globalNavigation'),
+		page = doc.getElementsByClassName('WikiaSiteWrapper')[0],
+		wrapper = doc.getElementById('WikiaTopAds');
+
+	function updateNavBar(height) {
+		var position = win.pageYOffset;
+
+		if (doc.body.offsetWidth <= breakPointWidthNotSupported) {
+			return;
+		}
+
+		if (position > height) {
+			wrapper.classList.remove('bfaa-pinned-nav');
+			nav.classList.remove('bfaa-pinned');
+		} else {
+			wrapper.classList.add('bfaa-pinned-nav');
+			nav.classList.add('bfaa-pinned');
+		}
+	}
+
+	function show(params) {
+		var backgroundColor = params.backgroundColor ? '#' + params.backgroundColor.replace('#', '') : '#000',
+			height = params.height || 0;
+
+		page.classList.add('bfaa-template');
+		wrapper.style.background = backgroundColor;
+
+		updateNavBar(height);
+		doc.addEventListener('scroll', adHelper.throttle(function () {
+			updateNavBar(height);
+		}, 50));
```

Comment:
[We lazy-load ads](https://github.com/Wikia/app/blob/733df9f5d176ef07e614828e30c2e1d2805371a0/extensions/wikia/Spotlights/js/LazyLoadAds.js#L17) with `100ms` throttle ([with a suggestion of doing it better](https://github.com/Wikia/app/blob/733df9f5d176ef07e614828e30c2e1d2805371a0/extensions/wikia/Spotlights/js/LazyLoadAds.js#L16)), we [use default throttle](https://github.com/Wikia/app/blob/77d7dd5fed8c44c42b82279c4d1757e17982628b/extensions/wikia/AdEngine/js/WikiaAdHelper.js#L6) for example in FMR set to `250` and `250` is also a throttle value set in [our images lazy-loading](https://github.com/Wikia/app/blob/733df9f5d176ef07e614828e30c2e1d2805371a0/extensions/wikia/ImageLazyLoad/js/ImgLzy.module.js#L24). @fraszczakszymon, do you think `250` would be too slow here?